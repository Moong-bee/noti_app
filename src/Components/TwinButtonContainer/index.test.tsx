import React from 'react'
import { render, screen } from '@testing-library/react'

import { TwinButtonContainer } from './index'

describe('<TwinButtonContainer /.', () => {
  it('올바른 랜더링', () => {
    const { container } = render(<TwinButtonContainer />)

    const LButton = screen.getByText('LButton')
    const RButton = screen.getByText('RButton')

    expect(LButton).toBeInTheDocument()
    expect(RButton).toBeInTheDocument()

    expect(container).toMatchSnapshot()
  })
})
