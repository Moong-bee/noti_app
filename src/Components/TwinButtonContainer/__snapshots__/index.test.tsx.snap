// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TwinButtonContainer /. 올바른 랜더링 1`] = `
<div>
  <div
    class="sc-dlnjwi gouvym"
  >
    <div
      class="sc-bdnxRM fLtUii"
      type="underline"
    >
      <div
        class="sc-gtsrHT fjHGAp"
        type="underline"
      >
        LButton
      </div>
    </div>
    <div
      class="sc-bdnxRM jZSYQY"
      type="fill"
    >
      <div
        class="sc-gtsrHT mNCHF"
        type="fill"
      >
        RButton
      </div>
    </div>
  </div>
</div>
`;
